index.js

```diff
@@ -5,46 +5,48 @@ var through = require('through2');
 var minimatch = require("minimatch");
 var _ = require('lodash');
 
-function instrument(options) {
-  var Instrumenter = require(resolve.sync(options.instrumenter, {basedir: __dirname}));
+function filterFiles(options, files) {
   var excludePattern = options.exclude ? [].concat(options.exclude) : [''];
```

Comment:
From the browserify docs:

> If opts.global is true, the transform will operate on ALL files, despite whether they exist up a level in a node_modules/ directory. Use global transforms cautiously and sparingly, since most of the time an ordinary transform will suffice.

The transform used to instrument the sources was never registered as a global transform. I'm now using a transform to collect the list of files to instrument later. It should work exactly as before and produces the same results as before for me.

Maybe you can explain, where you think the behavior has changed. It would cause quite a bit of work if this was released as a major ...